dashboard/app/models/script.rb

```diff
@@ -259,7 +259,10 @@ def self.get_without_cache(id)
   def self.get_from_cache(id)
     return get_without_cache(id) unless self.should_cache?
 
-    self.script_cache[id.to_s] || get_without_cache(id)
+    self.script_cache.fetch(id.to_s) do
+      # Populate cache on miss.
+      script_cache[id.to_s] = get_without_cache(id)
```

Comment:
at this line `should_cache?` must be `true`, not `false` (due to the `return ... unless self.should_cache?`), unless i'm reading this wrong